Dockerfile

```
@@ -8,9 +8,9 @@ RUN mamba install --yes "boa" "conda-verify"
 # Build all custom recipes in one command. We build our own conda packages to simplify
 # the environment creation process, as some of them were only available in pypi.
 COPY recipes /recipes
-RUN for pkg in rmath traits niflow-nipype1-workflows nitransforms pybids; do \
-    conda mambabuild --no-anaconda-upload /recipes/$pkg && \
-    conda build purge; \
+RUN for pkg in rmath traits nipype niflow-nipype1-workflows sqlalchemy pybids nitransforms tedana templateflow niworkflows sdcflows smriprep fmriprep; do \
+        conda mambabuild --no-anaconda-upload --use-local /recipes/$pkg && \
+        conda build purge; \
     done
```

By the way, if you give `conda mambabuild` the entire recipes folder, it will just build all of them. So no need for the for loop
```suggestion
RUN conda mambabuild --no-anaconda-upload --use-local /recipes && \
        conda build purge
```

---

I think it might handle that automatically